osbs/build/build_requestv2.py

```diff
@@ -59,6 +59,8 @@ def set_params(self, **kwargs):
         :param flatpak: if we should build a Flatpak OCI Image
         :param flatpak_base_image: str, name of the Flatpack OCI Image
         :param reactor_config_map: str, name of the config map containing the reactor environment
+        :param reactor_config_override: str, name of the config map containing the
```

Comment:
This is not correct. `build_request.py` says:
```
:param reactor_config_override: str, serialized reactor config yaml
```
Which is also not correct.

It should be something like this instead:
```
:param reactor_config_override: dict, data structure for reactor config to be injected as an environment variable into build; when used, reactor_config_map is ignored.
```

Reply:
Fixed, thanks.